src/components/modals/ButtonsModal.tsx

```diff
@@ -12,6 +12,8 @@ import { ModalUi4 } from '../ui4/ModalUi4'
 export interface ButtonInfo {
   label: string
 
+  disabled?: boolean
```

Comment:
Rejected. The buttons modal is for getting simple yes / no style questions. If you need more complicated interactions, you need to build your own modal. It's not complicated.

Jon, you of all people should understand this. You have been campaigning for re-usable components that are designed for a single purpose (`ModalFilledTextInput` comes to mind). You have also been fighting against cheap hacks, where a UI is "close enough" so we just tweak it instead of building the right thing.

We don't need to turn the buttons modal into a dumpster fire for some one-off feature. Just take your reusable buttons components, your reusable text components, and our reusable themed modal, and do it the right way.